PhotoCollection/PhotoCollection/View Controllers/PhotoCollectionViewController.swift

```diff
@@ -87,3 +87,19 @@ class PhotoCollectionViewController: UICollectionViewController {
         }
     }
 }
+
+extension PhotoCollectionViewController: UICollectionViewDelegateFlowLayout {
+    
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, insetForSectionAt section: Int) -> UIEdgeInsets {
+        UIEdgeInsets(top: 10, left: 10, bottom: 10, right: 10)
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, minimumInteritemSpacingForSectionAt section: Int) -> CGFloat {
+        10
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
+        let width = collectionViewLayout.collectionViewContentSize.width
+        return CGSize(width: width / 3, height: width / 3)
```

Comment:
Something to consider: complete the width division in line on 102, then pass in a constant that does not have to be mutated as a parameter.  Your code is exactly what I would do, just something I would consider to change when reviewing my own code.